### PR TITLE
Add C# variant to example code

### PR DESCRIPTION
**Pull request by VinnNo**

### CODE REVIEW

**Comment by pycbouh** [comment]
Please use more descriptive commit messages and PR titles 🙂 Same for #4657.

**Comment by VinnNo** [comment]
Hey, sorry about those @pycbouh! I was editing directly via github and it doesn't seem to allow me to change the PR names, it's automatically setting that with 'name-patch-[increment number]. I'll switch to CLI.

**Comment by pycbouh** [comment]
Oh no, I'm not talking about your branch names, those are irrelevant to the PR. I'm talking about the title, "Add C# variant to example code". It is taken directly from the commit message. So it's commit messages that would need to be a bit clearer. Your other PRs have good commit messages, for example.

File: tutorials/ui/gui_containers.rst [hunk 1]
```diff
@@ -179,3 +179,29 @@ to its rect size:
...
+                }
+            }
+        }
+        
+        public void Set_Something()
+        {
+            // Some setting changed, ask for children re-sort
+            QueueSort();
+        }
+    }
```
Comment (Calinou) [inline tutorials/ui/gui_containers.rst:207]:
```suggestion
 .. code-tab:: csharp

    using Godot;
    using System;

    public class Container : Godot.Container
    {
        public override void _Notification(int what)
        {
            if (what == NotificationSortChildren)
            {
                // Must re-sort the children
                foreach (Control c in GetChildren())
                {
                    // Fit to own size
                    FitChildInRect(c, new Rect2(new Vector2(), RectSize));
                }
            }
        }
        
        public void Set_Something()
        {
            // Some setting changed, ask for children re-sort
            QueueSort();
        }
    }
```

File: tutorials/ui/gui_containers.rst [hunk 2]
```diff
@@ -179,3 +179,30 @@ to its rect size:
...
+                // Must re-sort the children
+                foreach (Control c in GetChildren())
+                {
+                    // Fit to own size
+                    FitChildInRect(c, new Rect2(new Vector2(), RectSize));
+                }
+            }
+        }
+        
+        public void Set_Something()
```
Comment (paulloz) [inline tutorials/ui/gui_containers.rst:203]:
```suggestion
        public void SetSomeSetting()
```

File: tutorials/ui/gui_containers.rst [hunk 3]
```diff
@@ -179,3 +179,30 @@ to its rect size:
...
+    using System;
+
+    public class Container : Godot.Container
+    {
+        public override void _Notification(int what)
+        {
+            if (what == NotificationSortChildren)
+            {
+                // Must re-sort the children
+                foreach (Control c in GetChildren())
```
Comment (paulloz) [inline tutorials/ui/gui_containers.rst:195]: I know this variable is called `c` in GDScript too, but maybe we could change it and not endorse one letter variable naming? Doesn't seem like a good practice in general.

**Comment by mhilbrunner** [comment]
Same as https://github.com/godotengine/godot-docs/pull/4666 and the other PRs, this has been open for some time with unadressed feedback.

Closig, but its salvageable, so if you (or anyone else) wants to update this, feel free to ping for a reopen or to submit a new PR.

Thanks for contributing!

**Pull request closed without merging**